code/game/objects/effects/overlays.dm

```diff
@@ -132,6 +132,28 @@
 	duration = 8
 	randomdir = 0
 
+/obj/effect/overlay/temp/gib_animation
+	icon = 'icons/mob/mob.dmi'
+	duration = 15
+	randomdir = 0
```

Comment:
yeah, I forgot to remove that line, I added it to debug why the overlay weren't showing during my tests.